pages/common/cbt.md

```diff
@@ -0,0 +1,28 @@
+# cbt
+
+> Utility for reading data from Google Cloud's Bigtable.
+> More information: <https://cloud.google.com/bigtable/docs/cbt-reference>.
+
+- List tables in current project:
+
+`cbt ls`
+
+- Return count of rows in table:
```

Comment:
```suggestion
- Return count of rows in a specific table in the current project:
```

- Optional fix: `a specific` may sound better.
- Optional fix: it is possible to be explicit: as no project mentioned explicitly it is assumed that `the current` project is meant.